src/Shared/Recruit.Vacancies.Client/Infrastructure/Client/VacancyClient.cs

```diff
@@ -21,7 +20,7 @@
 
 namespace Esfa.Recruit.Vacancies.Client.Infrastructure.Client
 {
-    public class VacancyClient : IRecruitVacancyClient, IEmployerVacancyClient, IProviderVacancyClient, IJobsVacancyClient
+    public partial class VacancyClient : IRecruitVacancyClient, IEmployerVacancyClient, IJobsVacancyClient
```

Comment:
partial?

Reply:
I think this one has potential to become as big, so just watch the space grow. ;-)